Remove custom policies / clean up styles based on feedback from design

christiemolloy opened the pull request:
closes #88 #85 #84

## Prerequisites

- [X] Scope your styles to your individual card
- [X] You are using translations when necessary with proper plural/singular modifications
- [X] You are using the template card provided
- [X] Use functional components & hooks (please)
- [X] Attach screenshots (before and after)
- [X] Make sure that all text in blue is populated with the correct link. If you're unsure what the url should be, just ask!

## What

- Remove timestamp
- pie chart text shouldn't be changing
- compliance pie charts need to be same size as patch
- compliance pie charts change colors to match patch
- change font-size of bottom link in advisor
- moved styles out of `dashboard.scss` and into the relative card folders

## Screenshots

### Before
<img width="1427" alt="Screen Shot 2020-03-31 at 2 29 37 PM" src="https://user-images.githubusercontent.com/20118816/78062070-226d5f00-735c-11ea-91f2-ac7c2322ef7f.png">
<img width="1432" alt="Screen Shot 2020-03-31 at 2 29 41 PM" src="https://user-images.githubusercontent.com/20118816/78062078-25684f80-735c-11ea-8963-375471a2742e.png">

### After
<img width="1489" alt="Screen Shot 2020-03-31 at 2 24 24 PM" src="https://user-images.githubusercontent.com/20118816/78061749-abd06180-735b-11ea-903f-e9b18d5105cd.png">

<img width="1324" alt="Screen Shot 2020-03-31 at 2 24 35 PM" src="https://user-images.githubusercontent.com/20118816/78061756-aecb5200-735b-11ea-9770-4312db7f4871.png">


## Additional Info
[if needed]

## Code Review
@christiemolloy @ryelo @AllenBW

## Design Review
@kybaker

AllenBW commented on src/ChartTemplates/PieChart/PieChartTemplate.scss:
```diff
@@ -1,17 +1,9 @@
-    padding-top: 12px;
-    padding-left: 15px;
-}
+// .ins-c-pie-chart__legend {
```
an we kill the commented code (and file?)

AllenBW commented on src/PresentationalComponents/Dashboard/Dashboard.js:
```diff
@@ -70,11 +67,6 @@ const Dashboard = () => {
                         { permission.remediations ? <RemediationsCard /> : <DeniedState appName='Remediations'/> }
                     </Suspense>
                 </div>
-                <div className="dashboard-card-custom-policies">
```
farewell custom policies card, may you rest in peace 💀

AllenBW commented on src/SmartComponents/Compliance/ComplianceCard.js:
```diff
@@ -35,6 +36,7 @@ const ComplianceCard = ({ fetchCompliance, complianceFetchStatus, complianceSumm
 
     const intl = useIntl();
     const pieChartPadding = { bottom: 0, left: 0, right: 0, top: 0 };
+    const colorScale = ['#06c', '#519de9'];
```
can we use`@patternfly/react-tokens` here

AllenBW commented on src/SmartComponents/PatchManager/PatchManagerCard.js:
```diff
@@ -29,14 +29,14 @@ const PatchManagerCard = ({ systems, systemsStatus, fetchSystems, fetchSecurity,
         { x: intl.formatMessage(messages.securityAdvisories, { count: security }), y: security, fill: '#004b95' },
-        { x: intl.formatMessage(messages.bugfixAdvisories, { count: bugs }), y: bugs, fill: '#06c' },
-        { x: intl.formatMessage(messages.enhancementAdvisories, { count: enhancements }), y: enhancements, fill: '#519de9' }
+        { x: intl.formatMessage(messages.bugfixAdvisories, { count: bugs }), y: bugs, fill: '#519de9' },
```
ditto above,  use@patternfly/react-tokens here

kybaker commented:
@AllenBW @christiemolloy Can we reduce the height of the boxes in the top row. I suspect this was an accidental change. Otherwise looks great

christiemolloy commented:
hm I wasn't seeing any increase in height in the top row on my screen, thanks for pointing that out @AllenBW , I might need to add a max-height @kybaker

christiemolloy commented:
updated @AllenBW  , since I dont have my monitor im not able to stretch my screen to as tall as you have in the screenshot above, however I added a max-height to the first card group so that it wont grow any taller than what it is now (200px). Row 2 and 3 should continue to stretch as you have in the screenshot (it matches Kyles mocks). cc: @kybaker

AllenBW commented:
@christiemolloy if yah get to that other feedbacks, the number things, this'll look great!
